Add `Vocaloid Mapping Contest 2 & New Featured Artist: iroha(sasaki)` news post and associated wiki article (#10849)
ppy/osu-wiki: wiki/Contests/VMC/2/en.md

---
tags:
  - VMC2
  - VMC
---

# Vocaloid Mapping Contest 2

![VMC banner](/wiki/Contests/VMC/2/img/bannervmc.jpg)

The **Vocaloid Mapping Contest** (***VMC***) is a beatmapping contest hosted by the Vocaloid Appreciation Team, and is all about promoting VOCALOID, UTAU, and other vocal synth songs and artists that have fallen through the cracks. Mappers were tasked with creating the best beatmap possible, and then having them judged by some of the most prominent VOCALOID mappers in the community. 

This second iteration of the VMC featured a new VOCALOID Featured Artist, support for every game mode, and the return of a legendary VOCALOID mapper as a guest judge.

## Contest schedule

| Event | Timestamp |
| --: | :-- |

> | Announcement | 2024-01-27 |

Reviewer: ```suggestion
| Announcement | 2024-01-28 |
```

> | Submission | 2024-01-27 to 2024-03-01 |

Reviewer: ```suggestion
| Submission | 2024-01-28 to 2024-03-01 |
```

| Screening\* | 2024-03-01 to 2024-03-08 |
| Judging | 2024-03-08 to 2024-03-29 |
| Results livestream | 2024-03-31 @ 18:00 UTC (tentative) |

The results are streamed live on the [*Vocaloid Appreciation Team* Twitch account](https://www.twitch.tv/vocaloidappreciationteam) with commentary. Further details regarding the stream will be provided in this article, the news post, and in the Discord server.

\**Screening only occured in modes with a large number of entries.*

## Prizes

| Placing | Prize(s) |  |
| :-: | :-- | :-- |
| 1st place | Custom *VMC 2 Winner* VOCALOID plushie\* (made by [Noffy](https://osu.ppy.sh/users/1541323)), 6 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points), *VMC 2 Winner* profile badge | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
| 2nd place | 2 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |
| 3rd place | 1 month of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |

> \**Prize only for participants of the osu! game mode, due to it's one-of-a-kind nature.*

Reviewer 2: ```suggestion
\**Prize only for participants of the osu! game mode, due to its one-of-a-kind nature.*
```

- [Discord server](https://discord.gg/EruvDHaReY)

## Ruleset

- **Submissions must use one of the [selected songs](#songs).**
- **Submissions must be in `.osz` format.**
- **Beatmaps should abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes (such as an accidental off-screen object), submissions that contain intentional unrankables (not due to simple error, such as sliderator) will be disqualified. Mappers will be notified if this occurs.
- **Submissions must not be publicly revealed until after the results are announced.**
- **Excluding metadata, submissions must not include anything that could reveal your identity.** Metadata will be anonymised before submissions are relayed to judges and screeners.
- **Collaborations are not allowed.** This is a solo contest.
- **Submissions must not include storyboards or videos.**
- **Your submission must not have any edits to the `.osu` file that would not be possible through the editor.** This is not Aspire. You are permitted to change any difficulty settings, backgrounds, green lines, SV, hitsounds, etc.
- **Your submission must have only one difficulty.**

## Judges 

> | Game mode | Judge Names |

Reviewer: ```suggestion
| Game mode | Judges |
```

| osu!catch | ::{ flag=PH }:: [Jemzuu](https://osu.ppy.sh/users/7890134), ::{ flag=NL }:: [Greaper](https://osu.ppy.sh/users/2369776), ::{ flag=HK }:: [autofanboy](https://osu.ppy.sh/users/636114), ::{ flag=NL }:: [Chatie](https://osu.ppy.sh/users/6524765) | 
| osu!taiko | ::{ flag=US }:: [Alchyr](https://osu.ppy.sh/users/4993032), ::{ flag=UK }:: [aceticke](https://osu.ppy.sh/users/8838763), ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976), ::{ flag=US }:: [Lumenite-](https://osu.ppy.sh/users/6256027), ::{ flag=US }:: [Jonarwhal-](https://osu.ppy.sh/users/3653035) |
| osu!mania | ::{ flag=AU }:: [Decku](https://osu.ppy.sh/users/13360768), ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785), ::{ flag=US }:: [Monoseul](https://osu.ppy.sh/users/16010604), ::{ flag=DE }:: [BadDragon](https://osu.ppy.sh/users/6051688) |

## Judging criteria

| Position | Points | Description |
| :-: | :-: | :-: |

> | Relevance | 30 |  How well does the map fit the song? Do the elements of the map match the elements of the song, and is the map fundamentally inseparable from the song it is paired with? |

Reviewer: ```suggestion
| Relevance | 30 | How well does the map fit the song? Do the elements of the map match the elements of the song, and is the map fundamentally inseparable from the song it is paired with? |
```

| Overall Impression | 20 | The VOCALOID mappers participating as judges for this contest will all have personal biases with regard to how these songs are mapped. The judges will provide comments regarding the score they give. |
| Hitsounding | 5 | Rankable hitsounds can give up to 2 points, but hitsounds that fundamentally add to the experience and complement the map in a unique way can give the full 5 points. |

## Songs

Five (5) songs were selected by the staff from those provided by iroha(sasaki) for the purpose of this mapping contest. These songs were selected based on several factors, such as whether they have been ranked in the past, their suitability for the purposes of contest mapping, and other factors.

Please be sure to support iroha(sasaki).

| Song | VOCALOID | Lyricist | Length | BPM | Template |
| --: | :-: | :-: | :-: | :-: | :-: |
| The Cuckoo Bird's Beautiful Youth | VY1 & Kagamine Rin | π | 5:18 | ??? | [.osz]() |

> | Shirley!! | Kagamine Rin | π  | 4:59 | ??? | [.osz]() |

Reviewer: ```suggestion
| Shirley!! | Kagamine Rin | π | 4:59 | ??? | [.osz]() |
```

| Devolution | Kagamine Rin | kuma(alfred) | 4:26 | ??? | [.osz]() |
| 14 (Miku ver.) | Hatsune Miku | π | 4:55 | ??? | [.osz]() |

> | sigh | Kagamine Rin  | iroha(sasaki) | 4:08 | ??? | [.osz]() |

Reviewer: ```suggestion
| sigh | Kagamine Rin | iroha(sasaki) | 4:08 | ??? | [.osz]() |
```

## Contest schedule

| Event | Timestamp |
| --: | :-- |
| Announcement | 2024-01-28 |
| Submission | 2024-01-28 to 2024-03-01 |
| Screening[^screening] | 2024-03-01 to 2024-03-08 |
| Judging | 2024-03-08 to 2024-03-29 |
| Results livestream | 2024-03-31 @ 18:00 UTC (tentative) |

The results are streamed live on the [*Vocaloid Appreciation Team* Twitch account](https://www.twitch.tv/vocaloidappreciationteam) with commentary. Further details regarding the stream will be provided in this article, the news post, and in the Discord server.

## Prizes

| Placing | Prize(s) |  |
| :-: | :-- | :-- |
| 1st place | Custom *VMC 2 Winner* VOCALOID plushie[^plushie] (made by [Noffy](https://osu.ppy.sh/users/1541323)), 6 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points), *VMC 2 Winner* profile badge | ![](img/vmcwinnerbadge.png) |
| 2nd place | 2 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |
| 3rd place | 1 month of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |

All participating mappers receive constructive feedback from experienced mappers in the community.

## Links

> - [Announcement news post](https://osu.ppy.sh/home/news/)

Reviewer 2: ```suggestion
- [Announcement news post](https://osu.ppy.sh/home/news/2024-01-28-vocaloid-mapping-contest-2)
```

- [Contest listing](https://osu.ppy.sh/community/contests/)
- [Discord server](https://discord.gg/EruvDHaReY)

## Ruleset

> - **Submissions must use one of the [selected songs](#songs).**

Reviewer 3: comment from news file applies

| Creativity | 25 | How unique and outstanding is the map and the elements within it compared to the current mapping meta? How well does it stand out from the crowd? Does it evoke a feeling of "wow!", "that's cool!", or "that's super interesting!"? Generic mapping that copies the current mapping meta or concepts from other famous maps is discouraged. |
| Expertise | 20 | This refers to how well the map executes the elements it presents, encompassing the technical skills of the mapper, as well as the map's consistency, structure, and style. Judges will score this entry considering the overarching picture of the map, and will avoid nitpicking small details. |
| Overall Impression | 20 | The VOCALOID mappers participating as judges for this contest will all have personal biases with regard to how these songs are mapped. The judges will provide comments regarding the score they give. |
| Hitsounding | 5 | Rankable hitsounds can give up to 2 points, but hitsounds that fundamentally add to the experience and complement the map in a unique way can give the full 5 points. |

## Songs

> Five (5) songs were selected by the staff from those provided by iroha(sasaki) for the purpose of this mapping contest. These songs were selected based on several factors, such as whether they have been ranked in the past, their suitability for the purposes of contest mapping, and other factors.

Reviewer 3: ```suggestion
5 songs were selected by the staff from those provided by [iroha(sasaki)](https://osu.ppy.sh/beatmaps/artists/399) for the purpose of this mapping contest. These songs were selected based on several factors, such as whether they have been ranked in the past, their suitability for the purposes of contest mapping, and other factors.
```

> Please be sure to support iroha(sasaki).

Reviewer 3: ```suggestion
Please be sure to support [iroha(sasaki)](https://osu.ppy.sh/beatmaps/artists/399).
```

## Prizes

| Placing | Prize(s) |  |
| :-: | :-- | :-- |
| 1st place | Custom *VMC 2 Winner* VOCALOID plushie[^plushie] (made by [Noffy](https://osu.ppy.sh/users/1541323)), 6 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points), *VMC 2 Winner* profile badge | ![](img/vmcwinnerbadge.png) |
| 2nd place | 2 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |
| 3rd place | 1 month of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |

All participating mappers receive constructive feedback from experienced mappers in the community.

## Links

- [Announcement news post](https://osu.ppy.sh/home/news/2024-01-28-vocaloid-mapping-contest-2)
- [osu! contest listing](https://osu.ppy.sh/community/contests/199)
- [osu!taiko contest listing](https://osu.ppy.sh/community/contests/200)
- [osu!catch contest listing](https://osu.ppy.sh/community/contests/201)
- [osu!mania contest listing](https://osu.ppy.sh/community/contests/202)
- [Discord server](https://discord.gg/EruvDHaReY)

## Ruleset

> - **Submissions must use one of the [selected songs](#songs).** Cut versions of the songs are not allowed.

Reviewer 2: ```suggestion
- **Submissions must use one of the [songs from the selection below](#songs).** Cut versions are not allowed.
```

- **Collaborations are not allowed.** This is a solo contest.
- **Submissions must not include storyboards or videos.**
- **Your submission must not have any edits to the `.osu` file that would not be possible through the editor.** This is not Aspire. You are permitted to change any difficulty settings, backgrounds, green lines, SV, hitsounds, etc.
- **Your submission must have only one difficulty.**

## Judges

| Game mode | Judges |
| :-: | :-- |
| osu! | ::{ flag=CN }:: [NatsumeRin](https://osu.ppy.sh/users/151679), ::{ flag=US }:: [Noffy](https://osu.ppy.sh/users/1541323), ::{ flag=CN }:: [Mafumafu](https://osu.ppy.sh/users/3076909), ::{ flag=SV }:: [Natsu](https://osu.ppy.sh/users/1953876), ::{ flag=CA }:: [Shanipika](https://osu.ppy.sh/users/6336729), ::{ flag=NL }:: [Nyxa](https://osu.ppy.sh/users/3408126) |
| osu!catch | ::{ flag=PH }:: [Jemzuu](https://osu.ppy.sh/users/7890134), ::{ flag=NL }:: [Greaper](https://osu.ppy.sh/users/2369776), ::{ flag=HK }:: [autofanboy](https://osu.ppy.sh/users/636114), ::{ flag=NL }:: [Chatie](https://osu.ppy.sh/users/6524765) |

> | osu!taiko | ::{ flag=US }:: [Alchyr](https://osu.ppy.sh/users/4993032), ::{ flag=UK }:: [aceticke](https://osu.ppy.sh/users/8838763), ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976), ::{ flag=US }:: [Lumenite-](https://osu.ppy.sh/users/6256027), ::{ flag=US }:: [Jonarwhal-](https://osu.ppy.sh/users/3653035) |

Reviewer 4: ```suggestion
| osu!taiko | ::{ flag=US }:: [Alchyr](https://osu.ppy.sh/users/4993032), ::{ flag=GB }:: [aceticke](https://osu.ppy.sh/users/8838763), ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976), ::{ flag=US }:: [Lumenite-](https://osu.ppy.sh/users/6256027), ::{ flag=US }:: [Jonarwhal-](https://osu.ppy.sh/users/3653035) |
```
